src/Rainbow/Storage/SerializationFileSystemDataStore.cs

```diff
@@ -32,10 +32,10 @@ public SerializationFileSystemDataStore(string physicalRootPath, bool useDataCac
 			// ReSharper disable once DoNotCallOverridableMethodsInConstructor
 			PhysicalRootPath = InitializeRootPath(physicalRootPath);
 
-			_useDataCache = useDataCache;
+			UseDataCache = useDataCache;
```

Comment:
I tried that at first, but no go. Since the method I'm overriding is called in the constructor of the base class, both the base class constructor and my overridden method are executed before any code is hit in the derived instance constructor.